utils/themeswitch.js

```diff
@@ -0,0 +1,13 @@
+// DOM manipulation for switching color mode theme
+export const themeSwitch = theme => {
+	var root = document.documentElement

```

Comment:
Just a tiny change, I'd rather you use const here to make it consistent with the codebase.